medical-portal/src/UI/src/app/case-details/case-details.component.html

```diff
@@ -75,7 +75,7 @@ <h2>Active Case Details</h2>
                   </button>
                 </div>
               </div>
-              <div>
+              <div style="padding-top: 12px">
```

Comment:
at some point, maybe round 2, between the two of us, we should come up with a CSS standard. I noticed we are not using theme colors which is supported by SCSS/SASS and angular. Generally, inline CSS is considered bad but personally I'm on the fence.

Reply:
I agree Jebb. Inline styling is not the best practice i can change that to use the class.